client/my-sites/domains/domain-management/name-servers/custom-nameservers-form.jsx

```diff
@@ -118,7 +117,7 @@ class CustomNameserversForm extends React.PureComponent {
 					{ this.rows() }
 					{ this.popularHostsMessage() }
 
-					<FormFooter>
```

Comment:
I already asked in Slack, but for the sake of other reviewers, why did you opt to use a div instead of `FormFooter` here?

Reply:
The FormFooter applies additional styling to make the buttons appear on the right. I opted to stick with a `div` incase `FormFooter` is used in modals and to contain the buttons in such a way that other elements don't crash into them.